docs/docs/cmd/teams/tab/tab-add.md

```diff
@@ -53,3 +53,38 @@ Add teams tab for website with additional configuration which is unknown
 ```sh
 m365 teams tab add --teamId 00000000-0000-0000-0000-000000000000 --channelId 19:00000000000000000000000000000000@thread.skype --appId 06805b9e-77e3-4b93-ac81-525eb87513b8 --appName 'My Contoso Tab' --contentUrl 'https://www.contoso.com/Orders/2DCA2E6C7A10415CAF6B8AB6661B3154/tabView' --test1 'value for test1'
 ```
+
+## Response
+
+=== "JSON"
+
+    ``` json
+    {
+      "id": "8e454194-04c9-40aa-a9f3-7ab42d9541b5",
+      "displayName": "'Polly'",
```

Comment:
Let's try to align the output with the data used in the example. This case displayName would be `My Contoso Tab`. Same goes for text and csv.